src/main/java/com/fasterxml/jackson/dataformat/xml/XmlAnnotationIntrospector.java

```diff
@@ -183,5 +196,10 @@ public Boolean isOutputAsCData(Annotated ann) {
         public void setDefaultUseWrapper(boolean b) {
             // not used with JAXB
         }
+
+        @Override
+        public String getWrapperForIndexedType(AnnotatedClass ac) {
+            return JacksonXmlRootElement.DEFAULT_WRAPPER_NAME;
```

Comment:
I think I would rather return empty String, handle defaulting outside of this method; otherwise AI chaining will not work properly (as it is not possible to distinguish default value from explicit "item").